konfera/models/talk.py

```diff
@@ -51,6 +51,7 @@ class Talk(KonferaModel):
         blank=True,
         null=True,
     )
+    flag = models.CharField(max_length=32, blank=True)
```

Comment:
what about `null=True`? (maybe it's not needed, just making sure)

Reply:
In all other models for text fields, we use just blank=True, so I think this is fine since we are consistent with other models.